### PR TITLE
Register in Send To menu instead of plugins toolbar

### PR DESCRIPTION
**Pull request by mosseb**

### CODE REVIEW

**Comment by mosseb** [comment]
Need to update Ps icons

**Comment by mosseb** [comment]
Welln turns out the icons were already up to date.

File: main.qml [hunk 1]
```diff
@@ -5,8 +5,18 @@
...
+import QtQuick.Dialogs 1.0
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import AlgWidgets 2.0
+import AlgWidgets.Style 2.0
+
+import "photoshop.js" as Photoshop
 
 PainterPlugin {
+	property bool loading: false
```
Comment (PhilippeHermite) [inline main.qml:18]: Since you have an internal object, you should move this property inside it

**Review by PhilippeHermite — CHANGES_REQUESTED** [review]
You should update the plugin version and the min api in the plugin config
Aren't the translation missing?

**Pull request closed without merging**